app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt

```diff
@@ -135,6 +129,22 @@ fun SettingsViewContent(
         }
       }
       SettingsSectionDivider()
+      SettingsSection(
+        title = stringResource(R.string.settings_payments),
+        subTitle = stringResource(R.string.settings_powered)
+      ) {
+        Column(modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp)) {
+          SettingsCaretItem(
+            title = stringResource(R.string.settings_support_team),
+            onClick = onContactSupportClick
+          )
+          SettingsCaretItem(
+            title = stringResource(R.string.settings_billing),
+            onClick = onBillingTermsClick
+          )
+        }
+      }
+      SettingsSectionDivider()
```

Comment:
The divider is incorrect. according to figma it should not have a margin with the start/end of the screen